search-app/docs/api.md

```diff
@@ -1373,6 +1373,64 @@ Auto-completion assistance for building queries. This functionality may be used
 
 Collection facet autocompletion results are paged. See [Paging Details](#paging-details) for more information on how to page through autocomplete search results.
 
+#### Autocompletion of Science Keywords
+In the case of science keywords, the `fields` property may be used to determine the hierarchy of the term. The structure of the `fields` field is a colon (`:`) separated list in the following sequence:
+
+ * topic
+ * term
+ * variable-level-1
+ * variable-level-2
+ * varaible-level-3
+ * detailed-variable
+
+There may be gaps within the structure where no associated value exists.
+
+Example With variable-level-1 as the base term
+```
+{ :score 2.329206,
+  :type "science_keywords",
+  :value "Solar Irradiance",
+  :fields "Sun-Earth Interactions:Solar Activity:Solar Irradiance"
+}
+```
+
+Example with detailed-variable as the base term, note the extra colons preserving the structure
+```
+{ :score 1.234588,
+  :type "science_keywords",
+  :value "Coronal Mass Ejection",
+  :fields "Sun-Earth Interactions:Solar Activity:::Coronal Mass Ejection
```

Comment:
Are there supposed to be 5 colons?

Reply:
yes, thanks, I have updated the document.